scripts/dev-env.sh

```diff
@@ -22,9 +24,9 @@ GH_DEV_TOKEN_BW_ID=10242708-234a-4162-982f-ace700ee5c53
 write() {
     local name="$1"
     local value="$2"
-    sed -i "s/$name=.*/$name=$value/" "$target"
+    sed -i"" -e "s/$name=.*/$name=$value/" "$target"
```

Comment:
Have you checked this is portable between macOS (which uses BSD sed) and Linux?

Reply:
I'm on linux now and it works